global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "2.1.4"

```

Comment:
I think there are newer versions avaliable

Reply:
@forki I went with what the build script is doing. Should I update build.fsx too? I know the SAFE template is using 2.1.105